tied-buffer/index.md

```diff
@@ -1,201 +1,23 @@
-# Buffer tied to a single context
+# Buffer Tags
 
-| Proposal ID | CP008 |
+| Proposal ID | CP002 |
 |-------------|--------|
-| Name | Buffer tied to a single context |
-| Date of Creation | 14 March 2017 |
+| Name | Buffer Tags |
+| Previous Names | Buffer tied to a Context |
+| Date of Creation | 17 March 2017 |
 | Target | SYCL 2.2 |
 | Current Status | _Work In Progress_ |
 | Reply-to | Ruyman Reyes <ruyman@codeplay.com> |
 | Original author | Ruyman Reyes <ruyman@codeplay.com> |
-| Contributors | Gordon Brown <gordon@codeplay.com>, Mehdi Goli <mehdi.goli@codeplay.com> |
+| Contributors | Ruyman Reyes <ruyman@codeplay.com>, Gordon Brown <gordon@codeplay.com>, Victor Lomuller <victor@codeplay.com> |
 
 ## Overview
 
-This proposal aims to offer a buffer object that is tied to a specific SYCL 
-context, which allow  users to provide extra information to the SYCL 
-implementation to optimize usage for a single context and device.
-This concept vagely exists when using Shared Virtual Memory constructors
-on buffers, but it is defined as a specific use case for SVM buffers.
-With a specific `tied_buffer` class we aim to take advantage of
-the lower-overhead of a single-context buffer and simplify the 
-usage of SVM memory on SYCL 2.2
+This proposal aims to define an interface for specialising the buffer class whilst still allowing buffers of different types to be soted in a generic container, via the use of buffer tags and a common base class.
```

Comment:
Fixed, no if we go for the approach of having the tags as constructor parameters there's no need for a base class.